README.md

```diff
@@ -173,6 +187,27 @@ val rdd = sc.hbaseTable[(String, String)]("table")
 The example above retrieves all rows having a row key *greater or equal* to `00000` and *lower* than `00500`.
 The options `withStartRow` and `withStopRow` can also be used separately.
 
+It is also possible to filter by Time range and Timestamp
+
+### Time range
+
+```scala
+val rdd = sc.hbaseTable[(String, String)]("table")
+      .select("col")
+      .inColumnFamily(columnFamily)
+      .withTimerangeStartRow("1510679475915")
+      .withTimerangeEndRow("1510679475920")
```

Comment:
Why not just `withTimerangeStart` and `withTimerangeEnd`? The `Row` suffix seems a bit odd.